src/GraphQLParser.Tests/Visitors/ASTVisitorTests.cs

```diff
@@ -0,0 +1,60 @@
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using GraphQLParser.AST;
+using GraphQLParser.Visitors;
+using Shouldly;
+using Xunit;
+
+namespace GraphQLParser.Tests.Visitors;
```

Comment:
It's renamed `DefaultNodeVisitorTests.cs` , I only added `ASTVisitor_Should_Respect_CancellationToken`